### DISCUSSION TITLE
Vim: Move cursor to its last position

### DISCUSSION THREAD

**Discussion by thehaung** [JetBrains/ideavim#565] [Q&A]
Like the title I want to ask, Is VIM plugin have the feature like that?
This is stackoverflow question, i try it in native vim on linux and this worked. Also on Intellij is doesn't

Link: https://stackoverflow.com/questions/5052079/vim-move-cursor-to-its-last-position

**Comment by citizenmatt** [comment]
Which feature did you try? There are several mentioned on that page. There is support for marks and got jump lists, so please can you share a scenario and expected/actual steps?
